nodewatcher/nodewatcher.py

```diff
@@ -23,8 +23,15 @@
 import boto3
 import ConfigParser
 from botocore.config import Config
+import json
+import atexit
+import errno
+
 
 log = logging.getLogger(__name__)
+_NW_DATA_DIR = "/var/run/nodewatcher/"
```

Comment:
You can remove the `NW` prefix. Since this global var is only used in nodewatcher (as denoted by the `_` prefix) you don't need to indicate that it's for nodewatcher.